cephalopod/factory.py

```diff
@@ -66,6 +68,22 @@ def __call__(self, *args, **kwargs):
     return celery
 
 
+def get_config_path():
+    # env var has priority
+    try:
+        return os.path.expanduser(os.environ['CEPHALOPOD_CONFIG'])
+    except KeyError:
+        pass
+    # try finding the config in various common paths
+    paths = [os.path.expanduser('~/.cephalopod.conf'), '/etc/cephalopod.conf']
+    paths.insert(0, os.path.normpath(os.path.join(get_root_path('cephalopod'), 'cephalopod.conf')))
```

Comment:
@pferreir is taking care of the config file load in https://github.com/indico/cephalopod/pull/7